pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -844,22 +852,26 @@ func (qjm *XController) getAggregatedAvailableResourcesPriority(unallocatedClust
 		} else if value.Status.CanRun {
 			qjv := clusterstateapi.EmptyResource()
 			for _, genericItem := range value.Spec.AggrResources.GenericItems {
-				res, _ := genericresource.GetResources(&genericItem)
+				res, err := genericresource.GetResources(&genericItem)
+				if err != nil {
+					klog.Errorf("[getAggAvaiResPri] Failure getting resources for %s/%s, error: %#v, genericItem: %#v",
```

Comment:
There are several instances such as this one where the `genericItem` is shown on error. This can become noisy and probably not needed to be included in the logs. Opened to thoughts on this.

Reply:
When reviewing the logging noise I increased the log levels for standard info logs but for error logs I kept them the same because there is no way to change their level.

Is there any reason why there are several instances of this `genericItem` error?

Reply:
The instances are in different use cases, if I understand what you are asking. I can see 2 places where the `genericItem` is logged out with the error. I can remove those.